src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/task_4/model/Atm.java

```diff
@@ -0,0 +1,83 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects.task_4.model;
+
+public class Atm {
+    private int numberBanknotes100;
+    private int numberBanknotes50;
+    private int numberBanknotes20;
+    private int amount;
+
+    public Atm(int numberBanknotes100, int numberBanknotes50, int numberBanknotes20) {
+        this.numberBanknotes100 = numberBanknotes100;
+        this.numberBanknotes50 = numberBanknotes50;
+        this.numberBanknotes20 = numberBanknotes20;
+    }
+
+    public void addBanknotes100(int numberBanknotes100) {
+        this.numberBanknotes100 += numberBanknotes100;
+    }
+
+    public void addBanknotes50(int numberBanknotes50) {
+        this.numberBanknotes50 += numberBanknotes50;
+    }
+
+    public void addBanknotes20(int numberBanknotes20) {
+        this.numberBanknotes20 += numberBanknotes20;
+    }
+
+
+    public void isPossible(int amount) {
+        int totalSum = (this.numberBanknotes20 * 20) + (this.numberBanknotes50 * 50) + (this.numberBanknotes100 * 100);
+        if (amount <= totalSum) {
+            if (amount % 20 == 0 || amount % 50 == 0 || amount % 100 == 0) {
+                System.out.print("\nIs Possible");
+            } else {
+                System.out.print("\nIs not possible");
+            }
+        } else {
+            System.out.print("\nNo banknotes for this amount");
+        }
```

Comment:
This work, but it didn't work for everything data

Reply:
Do you mean that `amount` must be only positive? or something else?